api/location/schema.py

```diff
@@ -3,7 +3,7 @@
 
 from api.location.models import Location as LocationModel
 
-from api.room_resource.models import Resource as ResourceModel  # noqa: F401
+from api.devices.models import Devices as DevicesModel  # noqa: F401
```

Comment:
I think devices model is not necessary in the locations schema.

Reply:
It is as you are trying it is a declaration that you should nest till the `devices` model.